### DISCUSSION TITLE
Ocaml implementation

### DISCUSSION THREAD

**Discussion by leojrfs** [sqids/sqids-ocaml#2] [General]
Hey @4kimov!
Just a heads up, we are using Sqids and implemented [sqids for ocaml](https://github.com/hyper-systems/sqids-ocaml) for our use internally.

We will make it fully available and ready to merge with this repo very soon.

Cheers! 😀

**Comment by 4kimov** [comment]
Hey @leojrfs,

Thanks for the heads up! Looking forward to the PR, as I'm sure others will find this useful too 👍 

A few helpful tidbits on the PR here: https://github.com/sqids/sqids-spec?tab=readme-ov-file#-how-to-port-sqids-to-another-language

**Comment by 4kimov** [comment]
Repo code is there now thanks to @leojrfs! Closing discussion.
